### PR TITLE
sys-apps/ignition: bump to v0.19.0

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

File: sys-apps/ignition/ignition-9999.ebuild [hunk 1]
```diff
@@ -11,7 +11,7 @@ inherit coreos-go cros-workon systemd udev
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm64"
 else
-	CROS_WORKON_COMMIT="6a51b2f37ffec93c3a378578e14163f370585207" # tag v0.18.0
+	CROS_WORKON_COMMIT="b036c2d1a769bf4eca8b72cc3cd750d8f2fa5058" # tag v0.19.0
```
Comment (dm0-) [inline sys-apps/ignition/ignition-9999.ebuild]: What is this SHA1 from?  The tag is `01c039a5ce59acd39e5741713e59abfcb74d0782`.

Comment (ajeddeloh) [reply sys-apps/ignition/ignition-9999.ebuild]: Was on a different branch, didn't realize. Updated to be correct.

Comment (dm0-) [reply sys-apps/ignition/ignition-9999.ebuild]: It still has the bad value, unless GitHub is caching across page refreshes.

Comment (ajeddeloh) [reply sys-apps/ignition/ignition-9999.ebuild]: Sorry about that. Forgot to `git add` it.

**Pull request merged**